src/test/java/hudson/tasks/junit/JUnitResultArchiverTest.java

```diff
@@ -330,4 +338,40 @@ public String getName() {
         wc.waitForBackgroundJavaScript(10000L);
         assertThat(page.asText(), not(containsString(EXPECTED)));
     }
+
+    @Issue("JENKINS-26535")
+    @Test
+    public void testDescribableRoundTrip() throws Exception {
+        DescribableModel<JUnitResultArchiver> model = new DescribableModel<JUnitResultArchiver>(JUnitResultArchiver.class);
+        Map<String,Object> args = new TreeMap<String,Object>();
+
+        args.put("testResults", "**/TEST-*.xml");
+        JUnitResultArchiver j = model.instantiate(args);
+        assertEquals("**/TEST-*.xml", j.getTestResults());
+        assertFalse(j.isAllowEmptyResults());
+        assertFalse(j.isKeepLongStdio());
+        assertEquals(1.0, j.getHealthScaleFactor(), 0);
+        assertTrue(j.getTestDataPublishers().isEmpty());
+        assertEquals(args, model.uninstantiate(model.instantiate(args)));
+
+        // Test roundtripping of an instantiated list of test data publishers.
+        args.put("testDataPublishers", Collections.<TestDataPublisher>singletonList(new MockTestDataPublisher("test")));
```

Comment:
Anyway I would just delete this section. Passing raw `Describable`s as map values is deprecated—supported only for compatible with pre-1.0 Workflow scripts.